docs/BasicCommunication/OnSystemCapabilityUpdated/index.md

```diff
@@ -10,6 +10,9 @@ Purpose
 : Inform the HMI that a specific system capability has changed
 
 ### Notification
+1. HMI sends OnSystemCapabilityUpdated notification to SDL after creating a widget _(see also [CreateWindow](../../UI/CreateWindow/index.md))_.
```

Comment:
Above, the "sender" should be updated to be `SDL or HMI` to explain this RPC request is bi-directional.

Reply:
@JackLivio i beleive this one is fixed in c4098a746c102df39a042bae7085812a96493b0c